addon-test-support/index.js

```diff
@@ -1,11 +1,13 @@
-import QUnit from 'qunit';
+import require from 'require';
 import { hasRunAllDisposables } from 'ember-lifeline';
 
+let QUnit = require('qunit').default;
```

Comment:
this require should be done down inside of the `if (!setupTestDone) {` guard in `setupLifelineValidation` (which AFAICT is the only thing that is actually QUnit specific)

Also, if you update that to something like:

```js
if (!setupTestDone && require.has('qunit')) {
  let QUnit = require('qunit').default;
  QUnit.testDone(ensure....);
}
```